layers/+lang/typescript/config.el

```diff
@@ -1,2 +1,5 @@
 (defvar typescript-use-tslint t
-  "Use flycheck linter if not nil")
+  "Use flycheck linter if not nil.")
+
+(defvar typescript-fmt-on-save t
```

Comment:
Also I don't think format on save should be activated by default. It's a quite intrusive option, it should probably be activated in purpose.